appium/tests/specs/mock/setup/SetupShowsMeaningfulErrorFromFES.spec.ts

```diff
@@ -12,8 +12,7 @@ describe('SETUP: ', () => {
     await mockApi.withMockedApis(async () => {
       await SplashScreen.login();
       // todo - replace the following pause with wait for modal error
-      await browser.pause(5000);
+      await browser.pause(10000);
     });
```

Comment:
This one is unfinished. Else looks good

Reply:
@fcvakintos for finish this one - I need to create new method for check error, because we have different error type

Reply:
ok - please do.

Reply:
sure

Reply:
done